irony-completion.el

```diff
@@ -252,11 +268,12 @@ A candidate is composed of the following elements:
  2. The [result-]type of the candidate, if any.
  3. If non-nil, contains the Doxygen brief documentation of the
     candidate.
- 4. The signature of the candidate excluding the result-type
+ 4. The accessiblity of the candidate.
```

Comment:
By the way, regarding this function `irony-completion-candidates`.
I think it should be the one doing the filtering, IMHO you don't need the patch for company-irony (https://github.com/Sarcasm/company-irony/pull/32/).

The reason is that, the config is actually in irony-completion and not company-irony so it make sense to do the filtering here.
Secondly, it should also work for `irony-completion-at-point`
And finally, it is still possible for `company-irony` to introduce custom filtering just by let-binding the availability before calling this function.